web/massastation/src/utils/massaConstants.ts

```diff
@@ -5,3 +5,5 @@ export const massalabsNomination = ['Massa Labs'];
 export const PLUGIN_START = 'start';
 export const PLUGIN_STOP = 'stop';
 export const PLUGIN_UPDATE = 'update';
+
+export const isZipFile = /^(http)[^\s]*\.zip$/i;
```

Comment:
Shouldn't we prevent https also!? 🤔

Reply:
apparentky not we want https @Pierregvx  can u confirm ?

Reply:
https://github.com/massalabs/station/pull/803#discussion_r1234959509

Reply:
Ohh got it! It takes in account! Nevermind!